pyproject.toml

```diff
@@ -29,20 +29,20 @@ Documentation = "https://python-keycloak.readthedocs.io/en/latest/"
 "Issue tracker" = "https://github.com/marcospereirampj/python-keycloak/issues"
 
 [tool.poetry.dependencies]
-python = ">=3.7,<4.0"
-setuptools = "*"
+python = ">=3.8,<4.0"
+setuptools = ">=50.0.0"
```

Comment:
Is it still needed if you switched to `importlib`, not sure if it is used somewhere else too?

Reply:
True, I don't think it is, at least nothing was failing when it was removed during my testing. Thanks, will remove :)